Comparison Engine/Comparison Engine/Base Classes/BarManager.cs

```diff
@@ -78,7 +78,7 @@ public Bar GetBarByName(string barName)
             {
                 return barDictionary.Values.First(x => x.barName == barName);
             }
-            catch(ArgumentNullException e)
+            catch(Exception e)
             {
```

Comment:
why can't we catch like (ArgumentNullException | NullReferenceException e) or something similar, I can't remember what's the appropriate syntax. Catching Exception seems really fishy